integration-tests/rest-client-reactive/src/main/java/io/quarkus/it/rest/client/main/HelloClient.java

```diff
@@ -1,16 +1,29 @@
 package io.quarkus.it.rest.client.main;
 
 import javax.ws.rs.Consumes;
+import javax.ws.rs.NotFoundException;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import io.quarkus.rest.client.reactive.ClientExceptionMapper;
 
 @Path("")
 public interface HelloClient {
     @POST
     @Produces(MediaType.TEXT_PLAIN)
     @Consumes(MediaType.TEXT_PLAIN)
     String greeting(String name, @QueryParam("count") int count);
+
+    // this isn't used, but it makes sure that the generated provider can be properly instantiated in native mode
+    @ClientExceptionMapper
+    static RuntimeException toException(Response response) {
```

Comment:
can we use it instead? 
In the future we may have other native bugs in this and having half of the test already written maybe it's worth going all the way?

Reply:
Yeah, I'll certainly add the rest of the test in the future :)